src/clj/chronograph/db/organization.clj

```diff
@@ -16,11 +16,22 @@
                    :updated-at now}
                   db/sql-opts))))
 
+(defn where
+  ([options]
+   (where db/datasource options))
+  ([tx options]
+   (sql/find-by-keys tx :organizations options db/sql-opts)))
+
+(defn find [tx options]
+  (first (where tx options)))
+
 (defn find-by-slug
   ([slug]
    (find-by-slug db/datasource slug))
   ([tx slug]
-   (first (sql/find-by-keys tx
-                            :organizations
-                            {:slug slug}
-                            db/sql-opts))))
+   (find tx {:slug slug})))
+
+(defn by-ids [tx ids]
+  (sql/query tx
+             ["SELECT * FROM organizations where id = ANY(?)" (int-array ids)]
```

Comment:
`int-array` will blow up if it gets a collection of stringy ints and/or nils. 

So far it looks like by-ids gets called only in another db layer where it would get ints. 

But might make sense to distrust callers and be defensive here.

Reply:
I think it's okay to throw an exception here, in the DB layer. I think it should be the responsibility of domain layer to ensure the values match the spec. Do you feel we should add that logic there?

Reply:
At your discretion. It's just a personal preference to write defensive code when coercing types in Clojure.

If `by-ids` expected an int-array of ids, then I'd not check.